composer.json

```diff
@@ -13,9 +13,9 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "symfony/framework-bundle": "2.*|~3.0",
-        "symfony/yaml": "2.*|~3.0",
-        "google/apiclient": "1.*|2.*"
+        "symfony/framework-bundle": "2.*|~3.0|^4.0",
+        "symfony/yaml": "2.*|~3.0|^4.0",
+        "google/apiclient": "1.*|2.*|^2.0"
```

Comment:
2.* means `>=2.0.0 && <3.0.0`
^2.0 means `>=2.0.0 && <3.0.0`
Which is exactly the same. Please revert this change.

Reply:
Indeed, I totally agree with you, it's a careless mistake.. I reverted this !